packages/design-system-react/src/components/button-base/ButtonBase.test.tsx

```diff
@@ -181,12 +181,12 @@ describe('ButtonBase', () => {
 
     let button = screen.getByRole('button');
     expect(button).toBeDisabled();
-    expect(button).toHaveClass('opacity-50', 'cursor-not-allowed');
+    expect(button).toHaveClass('cursor-not-allowed');
 
     rerender(<ButtonBase isLoading>Loading Button</ButtonBase>);
     button = screen.getByRole('button');
     expect(button).toBeDisabled();
-    expect(button).toHaveClass('opacity-50', 'cursor-not-allowed');
+    expect(button).toHaveClass('cursor-not-allowed');
   });
```

Comment:
Updating all tests to check for the correct tailwind classnames for default button size of large and correct loading styles